<ui version="4.0" >
 <class>CreateZone</class>
 <widget class="QDialog" name="CreateZone" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Create a zone</string>
  </property>
  <property name="autoFillBackground" >
   <bool>true</bool>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="Name" >
     <property name="text" >
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QLineEdit" name="LEZoneName" >
     <property name="maxLength" >
      <number>32</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QGroupBox" name="TypeZone" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>340</width>
       <height>0</height>
      </size>
     </property>
     <property name="title" >
      <string>Type of zone</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QRadioButton" name="RBBox" >
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset>../../resources/zone_boxdxyz.png</iconset>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="RBSphere" >
        <property name="text" >
         <string/>
        </property>
        <property name="icon" >
         <iconset>../../resources/zone_spherepoint.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="QGroupBox" name="gBBox" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Coordinates</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="TLXmini" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>X mini</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QDoubleSpinBox" name="SpinBox_Xmini" >
        <property name="decimals" >
         <number>5</number>
        </property>
        <property name="maximum" >
         <double>999999999.000000000000000</double>
        </property>
        <property name="minimum" >
         <double>-999999999.000000000000000</double>
        </property>
        <property name="value" >
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="TLYmini" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Y mini</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QDoubleSpinBox" name="SpinBox_Ymini" >
        <property name="decimals" >
         <number>5</number>
        </property>
        <property name="maximum" >
         <double>999999999.000000000000000</double>
        </property>
        <property name="minimum" >
         <double>-999999999.000000000000000</double>
        </property>
        <property name="value" >
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="TLZmini" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Z mini</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QDoubleSpinBox" name="SpinBox_Zmini" >
        <property name="decimals" >
         <number>5</number>
        </property>
        <property name="maximum" >
         <double>999999999.000000000000000</double>
        </property>
        <property name="minimum" >
         <double>-999999999.000000000000000</double>
        </property>
        <property name="value" >
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="TLXmaxi" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>X maxi</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QLabel" name="TLZmaxi" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Z maxi</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QDoubleSpinBox" name="SpinBox_Zmaxi" >
        <property name="decimals" >
         <number>5</number>
        </property>
        <property name="maximum" >
         <double>999999999.000000000000000</double>
        </property>
        <property name="minimum" >
         <double>-999999999.000000000000000</double>
        </property>
        <property name="value" >
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="TLYmaxi" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Y maxi</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QDoubleSpinBox" name="SpinBox_Xmaxi" >
        <property name="decimals" >
         <number>5</number>
        </property>
        <property name="maximum" >
         <double>999999999.000000000000000</double>
        </property>
        <property name="minimum" >
         <double>-999999999.000000000000000</double>
        </property>
        <property name="value" >
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QDoubleSpinBox" name="SpinBox_Ymaxi" >
        <property name="decimals" >
         <number>5</number>
        </property>
        <property name="maximum" >
         <double>999999999.000000000000000</double>
        </property>
        <property name="minimum" >
         <double>-999999999.000000000000000</double>
        </property>
        <property name="value" >
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QGroupBox" name="gBSphere" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Coordinates</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="TLXcentre" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>X centre</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QDoubleSpinBox" name="SpinBox_Xcentre" >
        <property name="decimals" >
         <number>5</number>
        </property>
        <property name="maximum" >
         <double>999999999.000000000000000</double>
        </property>
        <property name="minimum" >
         <double>-999999999.000000000000000</double>
        </property>
        <property name="value" >
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="TLYcentre" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Y centre</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QDoubleSpinBox" name="SpinBox_Ycentre" >
        <property name="decimals" >
         <number>5</number>
        </property>
        <property name="maximum" >
         <double>999999999.000000000000000</double>
        </property>
        <property name="minimum" >
         <double>-999999999.000000000000000</double>
        </property>
        <property name="value" >
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="TLRayon" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Radius</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QDoubleSpinBox" name="SpinBox_Rayon" >
        <property name="decimals" >
         <number>5</number>
        </property>
        <property name="maximum" >
         <double>999999999.000000000000000</double>
        </property>
        <property name="minimum" >
         <double>-999999999.000000000000000</double>
        </property>
        <property name="value" >
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="TLZcentre" >
        <property name="sizePolicy" >
         <sizepolicy>
          <hsizetype>0</hsizetype>
          <vsizetype>0</vsizetype>
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Z centre</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QDoubleSpinBox" name="SpinBox_Zcentre" >
        <property name="decimals" >
         <number>5</number>
        </property>
        <property name="maximum" >
         <double>999999999.000000000000000</double>
        </property>
        <property name="minimum" >
         <double>-999999999.000000000000000</double>
        </property>
        <property name="value" >
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <widget class="QGroupBox" name="GBButtons" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string/>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QPushButton" name="buttonOk" >
        <property name="text" >
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonApply" >
        <property name="text" >
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonCancel" >
        <property name="text" >
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonHelp" >
        <property name="text" >
         <string>Help</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
